src/Form/AdminSettingsForm.php

```diff
@@ -206,9 +227,13 @@ private function buildFieldMappingsForm() {
     $field_mappings_settings = $this->config('acquia_lift.settings')->get('field_mappings');
     $field_names = $this->getTaxonomyTermFieldNames();
 
+    $taxonomy_vocabulary_url_text = t('Taxonomy Vocabulary');
+    $taxonomy_vocabulary_url = Url::fromRoute('entity.taxonomy_vocabulary.collection', [], ['attributes' => ['target' => '_blank']]);
+    $taxonomy_vocabulary_link = Link::fromTextAndUrl($taxonomy_vocabulary_url_text, $taxonomy_vocabulary_url)->toString();
+
     $form = [
       '#title' => t('Field Mappings'),
-      '#description' => t('Map Taxonomy vocabulary fields to "content section", "content keywords", and "persona" fields.'),
+      '#description' => t('Create ') . $taxonomy_vocabulary_link . t(' and map to "content section", "content keywords", and "persona" fields.'),
```

Comment:
Not fixed. Tried again and still failing in 0a3f1e3e360eb656ae7ff3d47c4b73b28e5987fd.

The code is fine and local test is fine. However, run-scripts.sh didn't like the placeholder when it [ran the tests](https://travis-ci.org/acquia/acquia_lift/jobs/165781971). Unless you know why, I will postpone the code update.